index.html

```diff
@@ -223,52 +141,84 @@ <h3>Skills</h3>
     <section class="section-contact" id="contact-section">
       <div class="desktop-contact d-flex">
         <p class="contact-desc">
-          I'm always interested in hearing about new projects, so if you'd like to
-          chat please get in touch.
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
         </p>
-        <form class="contact-form" action= "https://formspree.io/f/xqkoyalw" method="post">
+        <form
+          class="contact-form"
+          id="form"
+          action="https://formspree.io/f/xqkoyalw"
+          method="post"
+        >
           <label for="firstname">First name: </label>
-          <input name="name" maxlength="30" type="text" id="name" placeholder="Full name" required />
+          <span id="fullNameError"></span>
+          <input
+            name="name"
+            maxlength="30"
+            type="text"
+            id="name"
+            placeholder="Full name"
+          />
           <label for="email">Email: </label>
-          <input name="email" type="email" id="name" placeholder="Email address" required />
+          <span id="emailError"></span>
+          <input
+            name="email"
+            type="email"
+            id="email"
+            class="email"
+            placeholder="Email address"
+          />
           <label for="message">Message: </label>
```

Comment:
- [ ] There is a visible margin between the input elements and the text to the left. This is not the case with your design. Check screenshots below:
- Your design
![image](https://user-images.githubusercontent.com/62849872/232166029-045daec1-f882-4d3e-b16b-1f38c37d5ec0.png)
- Figma design
![image](https://user-images.githubusercontent.com/62849872/232166133-8857313f-0736-4038-803f-3aa75c91f339.png)